content/api/dashboards/code_snippets/result.api-dashboard-get.sh

```diff
@@ -0,0 +1,26 @@
+{
+  "dash": {
```

Comment:
This is an example for the legacy timeboard API: we need to change it with the new one 🙂

Reply:
oops. Thanks for catching that.